docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -895,7 +941,7 @@ public class TestSecurityAuthTest {
 }
 ----
 
-where `ProtectedResource` class may look like this:
+In the previous test code example, the `ProtectedResource` class might look like the following code snippet:
```

Comment:
`where` is a typical follow up when some code needs clarifications

Reply:
Updated per your following suggestion for this line to say "The `ProtectedResource` class which is used in this code example might look like this:"